src/templates/ratings/add.html

```diff
@@ -26,7 +26,7 @@
       </p>
       <p class="rating-label">{{ _('Please select a star rating.') }}</p>
       <textarea id="review-body" rows="2" cols="40" name="body"
-                placeholder="{{ _('Tell us what you love about this app!') }}"
+                placeholder="{{ _('Tell us what you like about this app. Write like you’re telling a friend about your experience with the app. Give specific details, for example, What features did you like or dislike?') }}" 
```

Comment:
We should probably add an L10n comment here to tell the localizers that this is the instruction for a text box.